cmd/dp-hierarchy-builder/main.go

```diff
@@ -2,127 +2,194 @@ package main
 
 import (
 	"context"
+	"github.com/ONSdigital/go-ns/server"
+	"github.com/gorilla/mux"
 	"os"
 	"os/signal"
 	"syscall"
 
 	"github.com/ONSdigital/dp-graph/graph"
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
 	"github.com/ONSdigital/dp-hierarchy-builder/config"
 	"github.com/ONSdigital/dp-hierarchy-builder/event"
 	"github.com/ONSdigital/dp-hierarchy-builder/hierarchy"
+	kafka "github.com/ONSdigital/dp-kafka"
 	"github.com/ONSdigital/dp-reporter-client/reporter"
-	"github.com/ONSdigital/go-ns/healthcheck"
-	"github.com/ONSdigital/go-ns/kafka"
-	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/log.go/log"
+)
+
+var (
+	// BuildTime represents the time in which the service was built
+	BuildTime string
+	// GitCommit represents the commit (SHA-1) hash of the service that is running
+	GitCommit string
+	// Version represents the version of the service that is running
+	Version string
 )
 
 func main() {
 	log.Namespace = "dp-hierarchy-builder"
-	log.Debug("Starting hierarchy builder", nil)
+	ctx := context.Background()
+
+	log.Event(ctx, "starting hierarchy builder", log.INFO)
 
 	cfg, err := config.Get()
-	if err != nil {
-		log.Error(err, nil)
-		os.Exit(1)
-	}
+	exitIfError(ctx, err, "error getting config")
 
 	// sensitive fields are omitted from cfg.String().
-	log.Debug("loaded config", log.Data{"cfg": cfg})
+	log.Event(ctx, "loaded config", log.INFO, log.Data{"cfg": cfg})
 
 	// a channel used to signal a graceful exit is required.
 	errorChannel := make(chan error)
```

Comment:
Can be removed as this is not used; I believe this has been replaced with apiErrors channel